backend/server/src/main/java/com/issuetracker/util/JwtUtil.java

```diff
@@ -39,7 +41,8 @@ public static UserDto decodeJwt(String token) {
             DecodedJWT jwt = verifier.verify(token);
             String login = jwt.getClaim(USER_LOGIN).asString();
             String name = jwt.getClaim(USER_NAME).asString();
-            return new UserDto(login, name);
+            String profileImageUrl = jwt.getClaim(USER_AVATAR_URL).asString();
```

Comment:
```suggestion
            String avatarUrl = jwt.getClaim(USER_AVATAR_URL).asString();
```

 기왕할거 용어가 통일되면 좋겠네요. 프론트분들께 건의를 해보죠